config/pre-install/v0.18.0/clusterrole.yaml

```diff
@@ -72,3 +72,16 @@ rules:
       - "delete"
       - "patch"
       - "watch"
+  - apiGroups:
+      - "flows.knative.dev"
+    resources:
+      - "parallels"
+      - "sequences"
+    verbs:
+      - "get"
```

Comment:
small nit, it'd probably be good to alias these commonly used verbs together, but lets snag that in a follow up PR